flask_security/decorators.py

```diff
@@ -63,7 +63,7 @@ def _get_unauthorized_view():
 
 
 def _check_token():
-    user = _security.login_manager.request_callback(request)
+    user = _security.login_manager.request_loader(request)
```

Comment:
`request_loader` should return `User` object but it returns `LocalProxy`. Because of that, while executing the next line it gives the error that below error.

> AttributeError: 'Request' object has no attribute 'is_authenticated'

If you replace `request_loader `  with `_request_callback` it gives proper output. But it's a wrong way because it is a static method.

Reply:
Actually - they made it python 'local' - and is called _request_callback now. I have requested that they restore that to public - which they did by providing some accessor properties. That is merged but not released.

Reply:
> Actually - they made it python 'local' - and is called _request_callback now. I have requested that they restore that to public - which they did by providing some accessor properties. That is merged but not released.

Any ETA for release.